hack/macros.py

```diff
@@ -1,6 +1,40 @@
 import os
+import semver
+import sys
+import traceback
+
+from github import Github
+
+# By default mkdocs swallows print() messages from macros
+def print_to_stdout(*a):
+    print(*a, file = sys.stdout)
+
+class GithubReleases:
+    def __init__(self):
+        self.tags_for_repo = {}
+        self.g = Github(os.getenv("GITHUB_TOKEN"))
+
+    def latest_release(self, major_minor, org, repo):
+        key = f'{org}/{repo}'
+
+        if key not in self.tags_for_repo:
+            tags = []
+            for release in self.g.get_repo(key, lazy=True).get_releases():
+                tags.append(release.tag_name)
+            tags = map(lambda tag: tag.removeprefix("knative-v"), tags)
+            self.tags_for_repo[key] = list(tags)
+
+        tags = self.tags_for_repo[key].copy()
+        tags = list(filter(lambda tag: tag.startswith(major_minor), tags))
```

Comment:
https://www.askpython.com/python/dictionary/sort-a-dictionary-in-python might want to just keep the sorted values in the dict itself. Before 3.6 this was done with a sorted dict type, but 3.6+ dicts are sorted types and thus remain sorted.

Reply:
One can sort by value as well, depends on lambda you pass along (https://datagy.io/python-sort-a-dictionary-by-values/), but I think since we are not really taking advantage of an ordered dict it is not worth the trouble and can be skipped.